lib/ex_unit/lib/ex_unit/diff.ex

```diff
@@ -67,10 +67,58 @@ defmodule ExUnit.Diff do
   def format(_left, _right, _formatter), do: nil
 
   defp format_string(string1, string2, formatter) do
-    String.myers_difference(string1, string2)
+    string_difference(string1, string2)
     |> Enum.map_join(&format_fragment(&1, formatter))
   end
 
+  defp string_difference(string1, string2) do
+    length1 = String.length(string1)
+    length2 = String.length(string2)
+    if bag_distance(string1, string2) / max(length1, length2) > 0.6 do
+      [del: string1, ins: string2]
```

Comment:
We can't return `nil` here as we call `format` recursively for inner data structures.
Alternative option is to move this check to the `ExUnit.Formatter` but only initial input will be verified.